oleggerasinovich/Titanic.ipynb

```diff
@@ -0,0 +1,1686 @@
+{
```

Comment:
<p>Speaking of visualizations, you can also show the distribution of the ages in each class using <code>df.plot.hist()</code>. ﻿</p><p> </p><p>E.g: <code>titanic.loc[titanic["Pclass"] == 3, "Age"].plot.hist()</code> . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/u2-de-students-lab/titanic_exploration/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>